### ISSUE TITLE
CompTIA rebranding Master/Expert Certs (CASP+ to SecurityX) + new Expert Level Certs

### ISSUE THREAD

**Issue by OMSKremer** [PaulJerimy/SecCertRoadmapHTML#151]
Hi, just for Info,

CompTIA rebrands the CASP+ Cert to "SecurityX" (Security eXpert) in December 2024. Old certificates are renamed and reissued with a new name.

There will also be two new Master/Expert Certs, DataX and CloudNetX (next Level for Network+ and Cloud+):

"CompTIA CloudNetX is an advanced networking certification. The certification will:
Validate skills in advanced network and system architecture for designing and managing complex, hybrid, IT infrastructures.
Be designed around the tasks performed by a network architect, infrastructure architect, enterprise architect and cloud architect.
Be a natural progression for the job roles aligned to CompTIA Network+ and CompTIA Cloud+"

Source: https://www.comptia.org/certifications/xpert-series

**Comment by smurfhunter** [comment]
I believe you've got a typo - CompTIA and ISACA are different entities entirely (unless I'm missing something? I checked both their websites, wikipedia and the internet and couldn't find a link between the two past offering certifications).

I don't think the datax will be applicable to cybersecurity - at least, it won't be primarily applicable, like the data+.

The cloudnetx has a lot of overlap with the CCNA and CCSP from what beta testers have been saying, so it will likely be somewhere around there. I suspect they're aiming to make a competitor to the CCNA with the certification, but that's speculation based off of community feedback and no actual fact.

SecurityX will just be a rebrand of the CASP+ so that will just need retitled on the page.

**Comment by OMSKremer** [comment]
> I believe you've got a typo - CompTIA and ISACA are different entities entirely (unless I'm missing something? I checked both their websites, wikipedia and the internet and couldn't find a link between the two past offering certifications).
> 
> I don't think the datax will be applicable to cybersecurity - at least, it won't be primarily applicable, like the data+.
> 
> The cloudnetx has a lot of overlap with the CCNA and CCSP from what beta testers have been saying, so it will likely be somewhere around there. I suspect they're aiming to make a competitor to the CCNA with the certification, but that's speculation based off of community feedback and no actual fact.
> 
> SecurityX will just be a rebrand of the CASP+ so that will just need retitled on the page.

you are right, i changed ISACA (i made my CISA last week ;)) to CompTIA, thanks.

**Comment by smurfhunter** [comment]
> you are right, i changed ISACA (i made my CISA last week ;)) to CompTIA, thanks.

All good, thank you for correcting it! Congratulations, as well!
